src/mlpack/methods/reinforcement_learning/q_networks/categorical_dqn.hpp

```diff
@@ -83,21 +81,21 @@ class CategoricalDQN
       vMax(config.VMax()),
       isNoisy(isNoisy)
   {
-    network.Add(new Linear<>(inputDim, h1));
-    network.Add(new ReLULayer<>());
+    network.Add(new mlpack::ann::Linear<>(inputDim, h1));
```

Comment:
I think in this file you can use `ann::` instead of `mlpack::ann`, since this is all in the `mlpack::rl` namespace (thus the `mlpack::` should be unnecessary).  Correct me if I'm wrong. :+1: